server/utilities/distance-utils.js

```diff
@@ -27,11 +27,19 @@ function degToRad(deg) {
  * Validates to ensure that the given lat/lons are valid.
  */
 function validateInputs(lat1, lon1, lat2, lon2) {
-    if (Math.abs(lat1) > 90 || Math.abs(lat2) > 90 || !lat1 || !lat2) {
+    if (isNaN(lat1) || isNaN(lat2) || !lat1 || !lat2) {
+        throw new Error("Input is not a number");
```

Comment:
I think this should say "Latitude is not a number" and the same change for longitude on line 43. It matches the fact that we throw different latitude/longitude invalid errors, and it's also more specific.